docs/userGuide/fullSyntaxReference.md

```diff
@@ -42,6 +42,7 @@
   tooltips : ['Tooltips', ['popups', 'reader-facing']],
   popovers : ['Popovers', ['popups', 'reader-facing']],
   modals : ['Modals', ['popups', 'reader-facing']],
+  annotations : ['Annotations', ['popups', 'reader-facing']],
```

Comment:
Wondering if it is better to put this under Images & Diagrams instead of Pop-ups?

Reply:
I was considering between the two as well, would like @damithc's thoughts on this.

Reply:
I think this is a better fit for images.